src/styles/forms.scss

```diff
@@ -129,34 +129,12 @@
 			}
 		}
 
-		.FlySelect_Option {
-			height: 30px;
-			padding: 0 10px;
-
-			.FlySelect__SecondaryText {
-				margin-right: 0;
-			}
-
-			&:hover {
-				color: $white;
-				background: $green-dark50;
-
-				span {
-					color: $white;
-				}
-
-				svg path {
-					fill: $white;
-				}
-			}
-		}
-
```

Comment:
This has been moved down below `.FlySelect_OptionGroup` for no reason other than it feels like this would be the order that they'd appear in.